lambdas/postgres-migration-count-tool/iam.tf

```diff
@@ -0,0 +1,125 @@
+data "aws_iam_policy_document" "migration_count_assume_role_policy" {
```

Comment:
This iam policy is nearly exact to the policy set from the proc lambda with a security group and/or access to the ECR in the event we want to run this via task.   We can remove the ECR statement if a Task isn't being utilized as a run method.